applying-stpa/openaps-level1-components.yml

```diff
@@ -0,0 +1,1322 @@
+        Text: Insulin pump clock is set to the wrong time (changing timezone and recalibrating
+          to local time, clocks changing, clock 'error' over time)
+      - Identifier: LS-2-UCA-STB-1-5
+        Text: OpenAPS clock is set to the wrong time (changing timezone and recalibrating
+          to local time, clocks changing, clock 'error' over time)
+      - Identifier: LS-2-UCA-STB-1-6
+        Text: OpenAPS has received inaccurate glucose data from the human to override
+          glucose data from NightScout
+      - Identifier: LS-2-UCA-STB-1-7
+        Text: The disk for RaspberryPi fills up and prevents the operating system
+          from running and completing its processes
+    - Identifier: UCA-STB-2
+      Text: OpenAPS does not set the temporary basal dose when there is no active
+        temporary basal recommendation and there is adequate data available to make
+        a justified recommendation
+      Reason: Not Providing
+      Scenarios:
+      - Identifier: LS-1-UCA-STB-2-1
+        Text: 'Flawed OpenAPS decision making algorithm (eg after pausing for a breakdown
+          it does not resume issueing doses) (UCA-STB-2: OpenAPS does not set the
+          temporary basal rate when there is no active temporary basal recommendation
+          and there is adequate data available to make a justified recommendation)'
+      - Identifier: LS-1-UCA-STB-2-2
+        Text: OpenAPS rig components are not connected together properly
+      - Identifier: LS-1-UCA-STB-2-3
+        Text: OpenAPS rig has run out of power
+      - Identifier: LS-1-UCA-STB-2-4
+        Text: OpenAPS rig dependencies / software has been updated and OpenAPS now
+          lacks compatibiliity with the insulin pump
+      - Identifier: LS-1-UCA-STB-2-5
+        Text: OpenAPS insulin dose calculation algorithm taking too long
+      - Identifier: LS-1-UCA-STB-2-6
+        Text: OpenAPS has not been configured correctly
+      - Identifier: LS-1-UCA-STB-2-7
+        Text: OpenAPS rig is hacked by an imposter
+      - Identifier: LS-1-UCA-STB-2-8
+        Text: OpenAPS clock is set to the wrong time (changing timezone and recalibrating
+          to local time, clocks changing, clock 'error' over time)
+      - Identifier: LS-1-UCA-STB-2-9
+        Text: OpenAPS bug causes it to process data from the insulin pump incorrectly
+          such that OpenAPS believes its recommendation is still active when informed
+          that it's not)
+      - Identifier: LS-1-UCA-STB-2-10
+        Text: 'Flawed OpenAPS decision making algorithm (either accidently or because
+          the human engineer thinks that more information is needed to make a dosing
+          recommendation than is actually necessary) (UCA-STB-2: OpenAPS does not
+          set the temporary basal rate when there is no active temporary basal recommendation
+          and there is adequate data available to make a justified recommendation)'
+      - Identifier: LS-2-UCA-STB-2-1
```

Comment:
We should still consider whether we want the identifiers to be like this: do we want `LS-1-UCA-....` and `LS-2-UCA-....` or is it better (easier to review and more aligned with common practices) if they're all in one list for each UCA?